File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
        int n;
        do
        {
            n = get_int("Height: ");
        }while(n<1);
        printf("\n");
        int k = n-1;
        for (int i = 0; i<n; i++)
        {
            for(int j = 1; j<k+1; j++)
            {
                printf(" ");
            }
            for(int l = 0; l<n-k; l++)
            {
                printf("#");
            }
            printf(" ");
            for(int l = 0; l<n-k; l++)
            {
                printf("#");
            }
            
            printf("\n");
            k--;
        }
}

File: mario.py
from cs50 import get_int
#getting a valid height
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9 :
        break
#printing pyramid
for i in range(n):
    #left half
    print(" "*(n - (i + 1)), end = "")
    print("#"*(i + 1), end = "")
    print("  ", end = "")
    #right half
    print("#"*(i + 1), end = "")
    print() # newline